kind/variant.mod

```diff
@@ -0,0 +1,15 @@
+provisioners:
+  textReplace:
+    Dockerfile:
+      from: "ARG KIND_VERSION={{ .kind.previousVersion }}"
+      to: "ARG KIND_VERSION={{ .kind.version }}"
+    goss/goss.yaml:
+      from: "- {{ .kind.previousVersion }}"
+      to: "- {{ .kind.version }}"
+
+dependencies:
+  kind:
+    releasesFrom:
+      githubReleases:
+        source: kubernetes-sigs/kind
+    version: "> 1.0"
```

Comment:
What is the meaning of this version?

Reply:
@s10myk4 
This is a common way to specify the version to be fixed in dependency management tools.
In this case, the version of `kind` greater than `1.0` will be updated.